src/chttpd/src/chttpd_db.erl

```diff
@@ -252,6 +252,10 @@ handle_view_cleanup_req(Req, Db) ->
     send_json(Req, 202, {[{ok, true}]}).
 
 
+handle_partition_req(#httpd{method='GET',path_parts=[DbName, <<"_partition">>, Partition, <<"_info">>]}=Req, _Db) ->
```

Comment:
let's put this at `/dbname/_partition/partition` as a parallel to getting db info at `/dbname`